test/srt/run_suite.py

```diff
@@ -135,7 +135,8 @@ class TestFile:
         TestFile("test_full_deepseek_v3.py", 333),
     ],
     "per-commit-8-gpu-b200": [
-        # add more here
+        TestFile("../../sgl-kernel/tests/test_fp4_gemm.py", 60),
+        TestFile("../../sgl-kernel/tests/test_fp4_quantize.py", 60),
```

Comment:
Can we also add

- https://github.com/sgl-project/sglang/blob/main/python/sglang/test/test_fp4_moe.py

- https://github.com/sgl-project/sglang/blob/main/python/sglang/test/test_block_fp8_deep_gemm_blackwell.py

- https://github.com/sgl-project/sglang/blob/main/sgl-kernel/tests/test_moe_fused_gate.py

- https://github.com/sgl-project/sglang/blob/main/python/sglang/test/attention/test_trtllm_mla_backend.py